enable/savage/svg/pathdata.py

```diff
@@ -40,7 +40,7 @@ class CaselessPreservingLiteral(CaselessLiteral):
 
     def __init__(self, matchString):
         super().__init__(matchString.upper())
-        self.name = "'%s'" % matchString
+        self.set_name("'%s'" % matchString)
```

Comment:
this will obviously not work for `pyparsing` v < 3 and we cannot drop support for `pyparsing` v < 3 yet so this needs to be written to handle the different versions of `pyparsing` differently.